src/Entity/SnoUser.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\SnoUserRepository;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Security\Core\User\PasswordAuthenticatedUserInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+#[ORM\Entity(repositoryClass: SnoUserRepository::class)]
+class SnoUser implements UserInterface, PasswordAuthenticatedUserInterface
```

Comment:
```suggestion
class SnowUser implements UserInterface, PasswordAuthenticatedUserInterface
```

Reply:
This is a prefix, in my web host I only have one database, so I must prefix all my entities to be able to have several sites in my database